rust/doc/openapi.yml

```diff
@@ -232,6 +232,8 @@ paths:
                   $ref: "#/components/examples/scan_id"
         "400":
           description: "Bad Request body"
+        "406":
```

Comment:
I guess status code `403` would be more fitting. According to https://en.wikipedia.org/wiki/List_of_HTTP_status_codes `406` is used, if the generated contend does not conform to the accept header, whereas `403` can be used if something is only allowed to exist once.